### PR TITLE
feat: randomize 'awesome' i18n text

### PR DESCRIPTION
**Pull request by pwltr**
### Description

Use a variety of words for 'Ok' buttons and choose one at random.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (improving code without creating new functionality)

### Tests

- [ ] Detox test
- [ ] Unit test
- [x] No test

### CODE REVIEW

File: src/utils/i18n/locales/en/common.json [hunk 1]
```diff
@@ -56,9 +56,11 @@
   "ok": {
     "string": "OK"
   },
-  "awesome": {
-    "string": "Awesome!"
-  },
+  "ok_random": {
+		"string": "Awesome! Nice! Cool! Great! Fantastic! Sweet! Excellent! Terrific!",
```
Comment (pwltr) [inline src/utils/i18n/locales/en/common.json]: I chose to do it this way so we don't have to have the same number of words for every lang

Comment (limpbrains) [reply src/utils/i18n/locales/en/common.json]: It is better to use line break as a separator, some languages may have OK phrase in 2 words

File: src/utils/i18n/locales/en/common.json [hunk 2]
```diff
@@ -56,9 +56,11 @@
...
+		"context": "A list of positive words to show the user after they have completed an action",
```
Comment (limpbrains) [inline src/utils/i18n/locales/en/common.json:61]: ❤️

**Review by limpbrains — APPROVED** [review]
Awesome! Cool! Заебца!

File: src/screens/Wallets/NewTxPrompt.tsx [hunk 3]
```diff
@@ -85,7 +86,7 @@ const NewTxPrompt = (): ReactElement => {
 					<View style={styles.buttonContainer}>
 						<Button
 							style={styles.button}
-							text={t('awesome')}
+							text={getRandomOkText()}
```
Comment (limpbrains) [inline src/screens/Wallets/NewTxPrompt.tsx:89]: this will generate new OK on every component re-render. Is it ok?

Comment (pwltr) [reply src/screens/Wallets/NewTxPrompt.tsx:89]: I think it's fine none of the screens where this is used should render more than once.

**Pull request merged**